### PR TITLE
Issue 57

### LINKED ISSUE #57: Test data
Consider creating sample data to test ETL methods. If we don't go this route, we should clean up the current test data

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #57

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -30,3 +30,4 @@ coverage = "*"
 coveralls = "*"
 pytest-cov = "*"
 jupyterlab = "*"
+mock = "*"
```
Comment (jsstevenson) [inline Pipfile:33]: mock is part of the standard library right? this might not be necessary

File: gene/etl/ensembl.py [hunk 2]
```diff
@@ -53,13 +52,13 @@ def _download_data(self):
                                self._fn)
             logger.info('Successfully downloaded Ensembl data file.')
 
-    def _extract_data(self, *args, **kwargs):
+    def _extract_data(self, ens_dir=PROJECT_ROOT / 'data' / 'ensembl',
```
Comment (jsstevenson) [inline gene/etl/ensembl.py:55]: do you still need the default `ens_dir` value here as well as above (line 41)? I think in the other normalizers we define them once as class variables in __init__. I'm not sure if that messes up how you've set up the mock tests, but just a thought.

File: gene/etl/hgnc.py [hunk 3]
```diff
@@ -36,10 +36,10 @@ def __init__(self,
         self._fn = fn
         self._version = None
 
-    def _download_data(self, *args, **kwargs):
+    def _download_data(self, hgnc_data_dir=PROJECT_ROOT / 'data' / 'hgnc',
```
Comment (jsstevenson) [inline gene/etl/hgnc.py:39]: ditto comment above about repeating data dir definition

**Review by jsstevenson — APPROVED** [review]

File: Pipfile (hunk 1 above)
Comment (korikuzma) [reply Pipfile:33]: We need to install it. I got an error when I didn't install it.

File: gene/etl/ensembl.py (hunk 2 above)
Comment (korikuzma) [reply gene/etl/ensembl.py:55]: Nah, I was in a rush to try and get mock to work so just left it. Will update the source classes to include data dir.

File: Pipfile (hunk 1 above)
Comment (jsstevenson) [reply Pipfile:33]: Ahhh interesting. It's under `unittest.mock` starting with version 3.3 but they keep a version on PyPI for backwards compatibility.   

https://mock.readthedocs.io/en/latest/

**Pull request merged**